app.py

```diff
@@ -6,6 +6,7 @@
 
 from flask import Flask
 
+from helpers.auth_helper import auth_helper
```

Comment:
Please, provide an example in readme, how to be authorized with curl.

Reply:
Done.